u6-spas/src/App.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div id="app">
+    <div id="nav">
+      <router-link :to="{ name: 'Home' }">Home</router-link> |
+      <router-link :to="{ name: 'Colors' }">Colors</router-link> |
+      <router-link :to="{ name: 'Profile' }">Edit Profile</router-link> |
```

Comment:
ในโครงการจริง มีแนวโน้มว่าคุณจะมีหน้าอื่นเพื่อดูโปรไฟล์ ฉันมักจะตั้งชื่อเส้นทางของฉันตามอ็อบเจกต์ + การกระทำ โดยให้ชื่อออบเจกต์ก่อน เพื่อให้รวมกลุ่มเข้าด้วยกัน เช่น
On a real-project, it's likely you'll have another page to view the profile. I usually name my routes based on the object + action, with the object first to group them together, for example:

- ProfileView
- ProfileEdit
- ProfileList
- ...etc.